openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx

```diff
@@ -853,3 +874,130 @@ export const getUniqueArray = (count: number) =>
   [...Array(count)].map((_, index) => ({
     key: `key${index}`,
   }));
+
+export const getAllRowKeysByKeyName = <T extends Field | SearchIndexField>(
```

Comment:
I recommend to use other specific utils instead common utlis
TableUtils will be a good fit I belive